docs/linking_to_external_code/private.md

```diff
@@ -0,0 +1,75 @@
+## Private modules and repositories
+
+There maybe instances where you want to load a remote module that is located in
+a _private_ repository, like a private repository on GitHub.
+
+Deno supports sending bearer tokens when requesting a remote module. Bearer
+tokens are the predominate type of access token used with OAuth 2.0 and is
+broadly supported by hosting services (e.g. GitHub, Gitlab, BitBucket,
+Cloudsmith, etc.).
+
+### DENO_AUTH_TOKENS
+
+The Deno CLI will look for an environment variable named `DENO_AUTH_TOKENS` to
```

Comment:
What about just "DENO_AUTH" ?

Reply:
I think we might want to add `DENO_AUTH_CERTS` in the future too for mTLS, so I think we should stick with `DENO_AUTH_TOKENS`.

Reply:
Irrespective of future environment variables, `DENO_AUTH` could be quite ambiguous.  `DENO_AUTH_TOKENS` is explicit that this is some form of authentication tokens, not login credentials, etc.  I think it is better to be a bit more explicit in this case.